### ISSUE TITLE
"lessmsi x from to" in powershell does weird things

### ISSUE THREAD

**Issue by czechdude** [activescott/lessmsi#55]
I try to extract msi to a folder with

PS>lessmsi x "D:\path to .msi" "D:\destination\pa th"

and it extracts the msi file to D:\ and tries to lookup some file called "D:\destination\pa th" in the msi archive:

Extracting 'D:....msi' to 'D:\'.
File D:\destination\pa th" was not found in the msi.

I installed the lessmsi latest from chocolatey.org into my powershell.

**Comment by czechdude** [comment]
so the problem seems to be the space in the output dir...

**Comment by Bioruebe** [comment]
> so the problem seems to be the space in the output dir...

To clarify the issue:
First of all, you forgot the trailing backslash. The documentation says : The path to extract the files to MUST have trailing backslash or it will be treated as one of the file_names.

But even with the backslash it wouldn't work :D
The real problem is the `\"` parsing method of .NET. See [this StackOverflow question](http://stackoverflow.com/q/9287812) and the remarks section of [GetCommandLineArgs()](https://msdn.microsoft.com/en-us/library/system.environment.getcommandlineargs.aspx#Anchor_2) for more information.

Basically, when parsing the command line
`lessmsi x "D:\a.msi" "D:\destination\"`,
the args array contains `D:\destination"` for the destination path.

As a workaround you can use
`lessmsi x "D:\a.msi" "D:\destination"\`,
which is wrong syntax, but it works.

**Comment by activescott** [comment]
this was fixed in 1.5.1. See https://github.com/activescott/lessmsi/releases/tag/v1.5.1
